pages/visualizations/overview/pr_over_time.py

```diff
@@ -109,30 +102,43 @@ def toggle_popover_7(n, is_open):
 # callback for prs over time graph
 @callback(
     Output("prs-over-time", "figure"),
-    Output("prs-over-time-timer", "n_intervals"),
     [
         Input("repo-choices", "data"),
-        Input("prs-over-time-timer", "n_intervals"),
         Input("pr-time-interval", "value"),
     ],
 )
-def prs_over_time_graph(repolist, timer_pings, interval):
-    logging.debug("IOT - PONG")
+def prs_over_time_graph(repolist, interval):
+
+    num_repos = len(repolist)
+    cache = cm()
+    ready = cache.existsm(func=prq, repos=repolist) == num_repos
 
-    ready, results, graph_update, interval_update = handle_job_state(jm, prq, repolist)
-    if not ready:
-        return graph_update, interval_update
+    while not ready:
+        time.sleep(1.0)
+        ready = cache.existsm(func=prq, repos=repolist) == num_repos
 
-    logging.debug("PRS_OVER_TIME_VIZ - START")
     start = time.perf_counter()
+    logging.debug("PULL REQUEST STALENESS - START")
+
+    # get all results from cache
+    results = cache.getm(func=prq, repos=repolist)
 
-    # create dataframe from record data
-    df = pd.DataFrame(results)
+    # deserialize results, create list of dfs
+    dfs = []
```

Comment:
why is this different than the others?

Reply:
missed it on refactor, thought I got all of them